### ISSUE TITLE
LSP Crash on opening any files

### ISSUE THREAD

**Issue by Myriad-Dreamin** [nvarner/typst-lsp#175]
- **Component**: 
    - [x] VSCode Extension
    - [ ] VSCodium Extension
    - [x] LSP (used with other editor)
- **Extension version**: 0.7.0
- **LSP version**: 0.7.0
- **OS version and name**: Windows 11

- [x] I am on the [latest](https://github.com/nvarner/typst-lsp/tags) stable version of the extension/LSP.
- [x] I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.

## Issue

When opening a file, it crashes at once.

### Logs

```
[Info  - 1:01:54 PM] Folder added to workspace: /path/to/workspace/
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src\workspace\source_manager.rs:180:61
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[Error - 1:01:54 PM] The Typst Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
```

**Comment by Myriad-Dreamin** [comment]
I cannot set "RUST_BACKTRACE=full" for a VSCode process so no full bracktrace :(.
Edited `AppData\Local\Programs\Microsoft VS Code\bin\code.cmd` with appending `set RUST_BACKTRACE=full`.

**Comment by BeitianMa** [comment]
I have encountered the same situation, restarting VSCode and reinstalling LSP did not work, looking for a solution.

**Comment by BeitianMa** [comment]
As a temporary solution, I rolled back the LSP version to 0.62 on VSCode and found that it worked fine.

**Comment by nvarner** [comment]
Is anyone who is having this issue able to share more details about the files they're working with? I'll investigate, but I haven't had this happen locally.

**Comment by nvarner** [comment]
This should be fixed in the latest version (v0.7.1) of the extension. If not, please reopen this issue.
